modules/tile-converter/src/3d-tiles-converter/helpers/gltf-converter.ts

```diff
@@ -34,30 +48,38 @@ export default class B3dmConverter {
    */
   async convert(
     i3sAttributesData: I3SAttributesData,
-    featureAttributes: any = null
+    featureAttributes: FeatureAttribute | null = null,
```

Comment:
The type `FeatureAttribute` is not correct. The loader returns `I3STileAttributes`

Reply:
Done.